### PR TITLE
Libc math: Refactoring

### PR DESCRIPTION
**Pull request by lokeshbv**
1)Typo fixes.
2)Fix for sqrtf.

### CODE REVIEW

**Comment by lokeshbv** [comment]
@sunghan-chang, Kindly review the changes.

File: lib/libc/math/lib_sqrt.c [hunk 1]
```diff
@@ -60,7 +60,9 @@
 #ifdef CONFIG_HAVE_DOUBLE
 double sqrt(double x)
 {
-	long double y, y1;
+	double y, y1;
```
Comment (sunghan-chang) [inline lib/libc/math/lib_sqrt.c]: [R12] Each variable shall be declared in the new line.
[Example-Do]
```
void example(void)
{
    int i;
    int index;
```bash
[Example-Don't]
```
void example(void)
{
    int i, index;
```bash

File: lib/libc/math/lib_sqrtf.c [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
 float sqrtf(float x)
 {
-	float y;
+	float y, y1;
```
Comment (sunghan-chang) [inline lib/libc/math/lib_sqrtf.c]: same as previous

**Comment by sunghan-chang** [comment]
@lokeshbv Please describe details. What do you update them? We can not know what you modify from commit message.

**Comment by lokeshbv** [comment]
@sunghan-chang  Details are updated.

**Comment by lokeshbv** [comment]
@sunghan-chang please review the latest commit also

**Pull request merged**